### PR TITLE
Make supply raids require 100% infil

### PR DESCRIPTION
**Pull request by Grobobobo**
Well, make ALL corpse missions that aren't invasions require 100% infil, As compensation, some of these missions have better detection rates. Seems experimental, but worth a try.

### CODE REVIEW

**Comment by BlackRose39** [comment]
My only concern is that what happens if you don't have enough time to infiltrate to 100%? Will the mission be unlaunchable?

**Comment by Grobobobo** [comment]
It should give you an option to either boost infiltration, or abort. Should is the key word here though.

**Comment by pledbrook** [comment]
I'll want to test this before merging.

File: LongWarOfTheChosen/Config/XComLW_InfiltrationSettings.ini [hunk 1]
```diff
@@ -60,6 +60,14 @@ RequiredInfiltrationToLaunch=-0.0000001f
 +MissionsRequiring100Infiltration=AdventFacilityFORGE_LW
 +MissionsRequiring100Infiltration=AdventFacilityPSIGATE_LW
 +MissionsRequiring100Infiltration=CentralNetworkBroadcast_LW
++MissionsRequiring100Infiltration=SupplyLineRaidATT_LW
++MissionsRequiring100Infiltration=SupplyLineRaidTrain_LW
++MissionsRequiring100Infiltration=SupplyLineRaidConvoy_LW
++MissionsRequiring100Infiltration=TroopManeuvers_LW
++MissionsRequiring100Infiltration=SecureUFO_LW
```
Comment (pledbrook) [inline LongWarOfTheChosen/Config/XComLW_InfiltrationSettings.ini]: I'd like to limit this to just Supply Raids for now.

**Comment by Grobobobo** [comment]
Definitely.

**Pull request closed without merging**